### DISCUSSION TITLE
YouTube video about FiraCode in Xcode / Mac Terminal

### DISCUSSION THREAD

**Discussion by rebeloper** [tonsky/FiraCode#1193] [Show and tell]
Hi,

I like Fira Code so much! I've been using it for years. All my latest YouTube tutorials are with Fira Code. So I decided to create a tutorial on how to add it to your Xcode project / Mac Terminal. Take a look at it here:
https://www.youtube.com/watch?v=fUk9hLLPvqo&list=PL_csAAO9PQ8Zia0zTZZjd0KZBEkcKb4yp&index=3

If you wish you may add this link with the image below to the README or the installation guide.
![DarkXcodeTheme](https://user-images.githubusercontent.com/2488011/106296985-f57b2880-625a-11eb-8808-7493a4b2b202.png)

Let me know what you think about the video. Any feedback is welcome. Thanks again for FiraCode!

**Comment by tonsky** [comment]
Great videos, thanks!
